<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover {
	background-color: rgb(186, 189, 182);
}
QPushButton{
	background-color: rgb(211, 215, 207);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_left">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_passenger">
         <property name="title">
          <string>Passenger</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>351</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <property name="verticalSpacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Passenger name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_name"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>weight in kg:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_weight"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="savePassenger">
               <property name="text">
                <string>Save Passenger</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_floor">
         <property name="title">
          <string>Floor</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>191</width>
            <height>89</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>current floor:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_floor"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Passenger:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_passenger_floor"/>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Embark</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>269</x>
            <y>30</y>
            <width>90</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_upAndDown">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_up">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>UP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_down">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>DOWN</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_building">
         <property name="title">
          <string>Building Control</string>
         </property>
         <widget class="QPushButton" name="pushButton_power">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>POWER OUT (toggle)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_fire_2">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>50</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>FIRE ALARM (toggle)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_nextTen">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>110</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Next 10 seconds</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_right">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Elevator</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>160</width>
            <height>244</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_others">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string>Inside
elevator 
buttons</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_help">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:orange</string>
             </property>
             <property name="text">
              <string>Help</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_fire">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:red</string>
             </property>
             <property name="text">
              <string>Fire</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_open">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Open &lt;&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Close &gt;&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>100</y>
            <width>160</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_destinations" rowstretch="0">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <property name="spacing">
            <number>5</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>137</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Elevator #</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_elevators"/>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>62</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Passenger:</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_passenger_elev">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>101</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>70</y>
            <width>92</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Disembark</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_sensor">
         <property name="title">
          <string>Sensor</string>
         </property>
         <widget class="QPushButton" name="pushButton_obstacle">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Door Obstacle (toggle)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_overweight">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>60</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Overweight (toggle)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_print">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Print Info</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
